### ISSUE TITLE
change port oscam

### ISSUE THREAD

**Issue by r100gs** [diddlesnaps/tvheadend#4]
Hello,

how can I change the oscam port?

Best regards,
Stefan

**Comment by diddledan** [comment]
This will be fixed by 824c5e6 when it is in the store

**Comment by diddledan** [comment]
you can now run the following to edit the configuration:

```bash
nano $(tvheadend.oscam-config)
```

Below is an example configuration to change the http port to 8080, and listen on every IP address assigned to the server:

```ini
[global]
serverip = 0.0.0.0

[webif]
httpport = 8080
```
